packages/pwa-kit-extension-sdk/src/configs/babel/utils.ts

```diff
@@ -31,21 +45,29 @@ const PLACEHOLDER_PATH = `${NODE_MODULES_PATH}/@salesforce/pwa-kit-extension-sdk
  * or symlinked paths that Babel doesn't support by using realpathSync.
  */
 export const buildBabelExtensibilityArgs = (config: any) => {
-    const extensions = getConfiguredExtensions(config)
-    const serverPath = fse.realpathSync(p.resolve(SERVER_PATH))
-    const placeHolderPath = fse.realpathSync(p.resolve(PLACEHOLDER_PATH))
+    try {
+        const extensions = getConfiguredExtensions(config)
+        const serverPath = fse.realpathSync(p.resolve(SERVER_PATH))
+        const placeHolderPath = fse.realpathSync(p.resolve(PLACEHOLDER_PATH))
 
-    const extensionSrcPaths =
-        extensions.length > 0
-            ? extensions.map(
-                  ([packageName]) =>
-                      fse.realpathSync(p.resolve(`${NODE_MODULES_PATH}/${packageName}/src`)) + '/**'
-              )
-            : []
+        const extensionSrcPaths =
+            extensions.length > 0
+                ? extensions.map(
+                      ([packageName]) =>
+                          fse.realpathSync(
+                              p.resolve(p.join(NODE_MODULES_PATH, packageName, 'src'))
+                          ) + '/**'
```

Comment:
should this be `\**` in window?

Reply:
No, since it's a glob pattern it works for both Windows and Unix.